packages/cli-e2e/__tests__/angular.specs.ts

```diff
@@ -0,0 +1,79 @@
+import retry from 'async-retry';
+
+import type {ChildProcessWithoutNullStreams} from 'child_process';
+import type {HTTPRequest, Browser, Page} from 'puppeteer';
+
+import {setupUIProject, teardownUIProject} from '../utils/cli';
+import {getNewBrowser} from '../utils/browser';
+import {isSearchRequest} from '../utils/platform';
+
+describe('ui', () => {
+  describe('create:angular', () => {
+    let browser: Browser;
+    const cliProcesses: ChildProcessWithoutNullStreams[] = [];
+    // TODO: CDX-90: Assign a dynamic port for the search token server on all ui projects
+    const clientPort = '4200';
+    const projectName = 'angular-project';
+    const searchPageEndpoint = `http://localhost:${clientPort}`;
+    const tokenProxyEndpoint = `http://localhost:${clientPort}/token`;
+    let interceptedRequests: HTTPRequest[] = [];
+    let page: Page;
+
+    beforeAll(async () => {
+      browser = await getNewBrowser();
+      await setupUIProject('ui:create:angular', projectName, cliProcesses, {
+        flags: ['--defaults'],
+        timeout: 40e3,
+      });
+    }, 3e6);
+
+    beforeEach(async () => {
+      page = await browser.newPage();
+
+      page.on('request', (request: HTTPRequest) => {
+        interceptedRequests.push(request);
+      });
+    });
+
+    afterAll(async () => {
+      await browser.close();
+      await teardownUIProject(cliProcesses);
+    }, 5e3);
+
+    it('should contain a search page section', async () => {
+      await page.goto(searchPageEndpoint, {
+        waitUntil: 'networkidle2',
```

Comment:
What is `networkidle2` ?

Reply:
`networkidle2` - consider navigation to be finished when there are no more than 2 network connections for at least 500 ms.